Dockerfile

```
@@ -12,6 +12,7 @@ RUN \
    --mount=type=cache,target=/go/pkg \
    --mount=type=cache,target=/root/.cache/go-build \
    go build -o charon "${GO_BUILD_FLAGS}" .
+RUN echo "Built charon version=$(./charon version )"
```

```suggestion
RUN echo "Built charon version=$(./charon version)"
```
or is the space intentional??